src/runtime/analytics-service-test.js

```diff
@@ -246,6 +253,42 @@ describes.realWin('AnalyticsService', {}, env => {
       await analyticsService.getLoggingPromise();
       expect(loggedErrors.length).to.equal(0);
     });
+
+    describe('SwG Clearcut Service Experiment', () => {
+      /**
+       * @param {number} expectedEventCount
+       */
+      async function verifyExpectedEventCount(expectedEventCount) {
+        // This triggers an event.
+        eventManagerCallback({
+          eventType: AnalyticsEvent.UNKNOWN,
+          eventOriginator: EventOriginator.UNKNOWN_CLIENT,
+          isFromUserAction: null,
+          additionalParameters: null,
+        });
+
+        // These wait for analytics server to be ready to send data.
+        expect(analyticsService.lastAction_).to.not.be.null;
+        await analyticsService.lastAction_;
+        await activityIframePort.whenReady();
+
+        expectOpenIframe = true;
+        expect(eventsLoggedToService.length).to.equal(expectedEventCount);
```

Comment:
> I like having shared code when its identical - what do you think about the function I created instead?

I think this TotT makes some good points for readability over uniqueness in tests specifically
go/tott/598

Reply:
I mostly disagree with the advice.  I find it significantly more time consuming to figure out what dozens of tests do when they all do the same thing over and over again.  I think you are much more likely to miss the 1 random test that does something subtly different because of all the repeated code.  But this is just 2 tests and its not that important to me,